pywbem/mof_compiler.py

```diff
@@ -1897,8 +1897,15 @@ def DeleteQualifier(self, *args, **kwargs):
 class MOFWBEMConnection(BaseRepositoryConnection):
     """
     A CIM repository connection to an in-memory repository on top of an
-    underlying repository, that is used by the MOF compiler to provide rollback
-    support.
+    underlying repository API, that is used by the MOF compiler to provide
```

Comment:
I think with gaining more clarity about the nature of this construct, I suggest describing it as:
"A CIM repository connection to an underlying WBEM connection with an in-memory roll-back log on top of it, that provides the ability to roll back changes."